github/pulls_reviews_test.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestPullRequestsService_ListReviews(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/pulls/1/reviews", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypePullRequestReviewsPreview)
+		fmt.Fprint(w, `[{"id":1},{"id":2}]`)
+	})
+
+	reviews, _, err := client.PullRequests.ListReviews("o", "r", 1)
+	if err != nil {
+		t.Errorf("PullRequests.ListReviews returned error: %v", err)
+	}
+
+	want := []*PullRequestReview{
+		{ID: Int(1)},
+		{ID: Int(2)},
+	}
+	if !reflect.DeepEqual(reviews, want) {
+		t.Errorf("PullRequests.ListReviews returned %+v, want %+v", reviews, want)
+	}
+}
```

Comment:
Looks nice! Thanks :+1: https://github.com/google/go-github/pull/497/commits/8c7f0fb864466393e53815c0b7f9e127b824ce83

Could you please adding similar tests for the rest of methods for consistency?